tests/layer_tests/pytorch_tests/test_scaled_dot_product_attention.py

```diff
@@ -12,31 +12,37 @@ class TestScaledDotProductAttention(PytorchLayerTest):
     def _prepare_input(self):
         return (np.random.randn(1, 2, 8, 4).astype(np.float32), np.random.randn(1, 2, 8, 4).astype(np.float32), np.random.randn(1, 2, 8, 4).astype(np.float32))
 
-    def create_model(self, mask, is_causal):
+    def create_model(self, mask, is_causal, scale):
         import torch.nn.functional as F
         import torch
 
         class aten_scaled_dot_product_atten(torch.nn.Module):
 
-            def __init__(self, mask=False, is_causal=False) -> None:
+            def __init__(self, mask=False, is_causal=False, scale=False) -> None:
                 super().__init__()
 
-                self.mask = None if not mask else torch.from_numpy(np.random.randint(0, 2, (8, 8)).astype(np.float32))
+                self.mask = None if not mask else torch.from_numpy(
+                    np.random.randint(0, 2, (8, 8)).astype(np.float32))
                 self.is_causal = is_causal
                 if is_causal and mask:
                     self.mask.to(torch.bool)
                     self.is_causal = False
 
+                self.scale = None if not scale else torch.tensor(
+                    5, dtype=torch.float)
+
             def forward(self, query, key, value):
-                return F.scaled_dot_product_attention(query, key, value, attn_mask=self.mask, is_causal=self.is_causal)
+                return F.scaled_dot_product_attention(query, key, value, attn_mask=self.mask, is_causal=self.is_causal, scale=self.scale)
 
         ref_net = None
 
-        return aten_scaled_dot_product_atten(mask, is_causal), ref_net, "aten::scaled_dot_product_attention"
+        return aten_scaled_dot_product_atten(mask, is_causal, scale), ref_net, 'aten::scaled_dot_product_attention'
 
     @pytest.mark.nightly
     @pytest.mark.precommit
     @pytest.mark.precommit_fx_backend
```

Comment:
@cavusmustafa, can I run the test in "fx-mode" locally?

Reply:
We recently discovered a bug in fx unit tests and @suryasidd is currently working on a fix.